lib/Runtime/Language/InterpreterProcessOpCodeAsmJs.h

```diff
@@ -122,6 +122,15 @@
 #define PROCESS_D2toD1Mem(name, func) PROCESS_D2toD1Mem_COMMON(name, func,)
 #define PROCESS_F2toF1Mem(name, func) PROCESS_F2toF1Mem_COMMON(name, func,)
 
+#define PROCESS_EMPTYASMJS(name, func) \
```

Comment:
@Cellule  `PROCESS_CUSTOM_ASMJS_COMMON` always passes an arg to `func`. It's looks both ugly and confusing to have an argument that isn't actually used. If you prefer I can get rid of  `PROCESS_EMPTYASMJS` and use PROCESS_READ_LAYOUT_ASMJS or some other one-arg macro. 

```
        PROCESS_READ_LAYOUT_ASMJS(name, layout, suffix); \
        func(playout); \
        break; \
```